### PR TITLE
[NPG-3427, NPG-3367] Snapshot endpoints: "/api/v0/snapshot/voter/{tag}/{voting_key}:", "  /api/v0/snapshot/delegator/{tag}/{stake_public_key}:"

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
- Implement new endpoint `/api/v0/snapshot/delegator/{tag}/{stake_public_key}:` which returns a delegator's info by the provided stake public key
- Update `/api/v0/snapshot/voter/{tag}/{voting_key}:` endpoint, add new `voting_power_saturation` field, voting power's share of the total voting power corresponds to the current voting group.
- make some refactoring, renaming functions etc.

### CODE REVIEW

**Review by stevenj — APPROVED** [review]
API side of things looks ok to me.
Code looks ok to me to.

File: vit-servicing-station-lib/src/db/models/snapshot.rs [hunk 1]
```diff
@@ -65,24 +65,24 @@ pub struct Contributor {
     pub snapshot_tag: String,
 }
 
-impl Insertable<contributors::table> for Contributor {
+impl Insertable<contributions::table> for Contribution {
```
Comment (cameron1024) [inline vit-servicing-station-lib/src/db/models/snapshot.rs:68]: This looks derivable, can't see anything funky in the impl

**Review by cameron1024 — APPROVED** [review]
LGTM, just one small thing

File: vit-servicing-station-lib/src/db/models/snapshot.rs (hunk 1 above)
Comment (Mr-Leshiy) [reply vit-servicing-station-lib/src/db/models/snapshot.rs]: Sorry, was my fault. Yes you are right, updated

**Pull request merged**